src/editorial-source-components/InputHeading.tsx

```diff
@@ -20,12 +21,17 @@ const Errors = ({ errors }: { errors: string[] }) =>
 
 type Props = {
   label: React.ReactNode;
+  description?: React.ReactNode;
   errors: string[];
 };
 
-export const InputHeading = ({ label, errors }: Props) => (
+export const InputHeading = ({ label, description, errors }: Props) => (
   <InputHeadingContainer>
     <Label>{label}</Label>
-    <Errors errors={errors} />
+    {errors.length > 0 ? (
+      <Errors errors={errors} />
+    ) : description ? (
+      <Description>{description}</Description>
+    ) : null}
```

Comment:
Ah, I like this approach 👍 not quite convinced these will always be mutually exclusive, but let's find out 😁